src/components/MultiHopTrade/components/TradeConfirm/hooks/useThorStreamingProgress.tsx

```diff
@@ -72,101 +76,91 @@ export const useThorStreamingProgress = ({
   attemptedSwapCount: number
   totalSwapCount: number
   failedSwaps: StreamingSwapFailedSwap[]
-  numSuccessfulSwaps: number
+  successfulSwapCount: number
```

Comment:
i.e here we'd use `attemptedSwapCount` instead

Reply:
Used {attemptedSwapCount}/{maxSwapCount} then, with fallback when completed to {attemptedSwapCount/attemptedSwapCount}

It's simpler like that and would align with everything

Hopefully I didn't lost myself with all those changes...